src/module/chat/cards.ts

```diff
@@ -128,11 +129,7 @@ export class IronswornChatCard {
         stat,
       })
     } else {
-      const i18nKey = {
-        [ROLL_OUTCOME.STRONG_HIT]: 'Strong_hit',
-        [ROLL_OUTCOME.WEAK_HIT]: 'Weak_hit',
-        [ROLL_OUTCOME.MISS]: 'Miss',
-      }[hittype]
+      const i18nKey = ''
```

Comment:
Somewhere along the line, momentum-burn updates got broken. I think this is the culprit for the specific breakage:

![CleanShot 2022-08-22 at 06 21 50](https://user-images.githubusercontent.com/39902/185931789-c6f9705e-0df1-413e-9e2f-af5e20ecf863.jpg)